core/src/Kokkos_View.hpp

```diff
@@ -1749,6 +1749,20 @@ class View : public ViewTraits<DataType, Properties...> {
     m_map   = std::move(rhs.m_map);
     return *this;
   }
+  // -----------------------------------------------------------
+  // public is_assignable
+  // expose underlying view mapping property
+  template <class RT, class... RP>
+  inline const bool is_assignable(const View<RT, RP...>& rhs) {
```

Comment:
Why isn't this method `const`? And returning a `const bool` makes no sense; please just return `bool` here.

Reply:
method has been removed, but it was const because the first parameter of assignable_layout_check wasn't const...I also changed that to const when I removed this method and added the implementation for variant 2 above.